k8s-install/scripts/install_cni_test.go

```diff
@@ -143,38 +144,7 @@ var _ = Describe("install-cni.sh tests", func() {
 			It("Should parse and output a templated config", func() {
 				err := runCniContainer()
 				Expect(err).NotTo(HaveOccurred())
-
-				expectedFile := "expected_10-calico.conf"
-
-				var expected = []byte{}
-				if file, err := os.Open(expectedFile); err != nil {
-					Fail("Could not open " + expectedFile + " for reading")
-				} else {
-					_, err := file.Read(expected)
```

Comment:
expected was a 0-length slice at this point and `file.Read()`'s contract says that it reads at most `len(<input slice>)` bytes; hence it was reading 0 bytes!

Reply:
:-) And unnecessary anyway, given that they should use ioutil.ReadFile.